bsp/src/mill/bsp/BSP.scala

```diff
@@ -85,4 +73,18 @@ object BSP extends ExternalModule with CoursierModule {
     )
   }
 
+  private[mill] case class InstallArgs(jobs: Int = 1)
```

Comment:
I'd expect that you use this class in the `install` command?

Reply:
Just removed the original `BSP/install` command altogether (and this class too actually)